web_trackers/druple_inline_search_vC-2-10-2.js

```diff
@@ -0,0 +1,34 @@
+// <!-- Snowplow starts plowing - Standalone Search vB.2.10.2 -->
+;(function(p,l,o,w,i,n,g){if(!p[i]){p.GlobalSnowplowNamespace=p.GlobalSnowplowNamespace||[];
+    p.GlobalSnowplowNamespace.push(i);p[i]=function(){(p[i].q=p[i].q||[]).push(arguments)
+    };p[i].q=p[i].q||[];n=l.createElement(o);g=l.getElementsByTagName(o)[0];n.async=1;
+    n.src=w;g.parentNode.insertBefore(n,g)}}(window,document,"script","https://sp-js.apps.gov.bc.ca/MDWay3UqFnIiGVLIo7aoMi4xMC4y.js","snowplow"));
+var collector = 'spm.gov.bc.ca';
+window.snowplow('newTracker','rt',collector, {
+    appId: "Snowplow_standalone",
+    platform: 'web',
+    post: true,
+    forceSecureTracker: true,
+    contexts: {
+        webPage: true,
+        performanceTiming: true
+    }
+});
+window.snowplow('enableActivityTracking', 30, 30); // Ping every 30 seconds after 30 seconds
+window.snowplow('enableLinkClickTracking');
+window.snowplow('trackPageView');
+
+if (window.location.pathname.split('/')[1] == 'search') {
+    window.snowplow('trackSiteSearch', [decode_search(window.location.pathname.split('/')[2]])
+  }
+
+  function decode_search(terms) {
+    decoded = decodeURIComponent(terms);
+    
+    if ( decoded.indexOf("advanced")  >= 0 ) {
+       // This will find the bs= parameter and get the terms into an array. It also splits the "excat phrase" into separate terms.                                  
+      decoded = decoded.split(';')[5].slice(3).replace(/"/g,"").split(' ');
```

Comment:
Maybe consider if the chained methods should be split onto separate lines for readability?